docs/docs/integrations/promotions/voucherify.md

```diff
@@ -0,0 +1,90 @@
+
+# Voucherify Integration
+  
+Voucherify is an API-first Promotions and Loyalty Engine that helps brands run personalised coupons, gift cards, auto-applied promotions, loyalty programs, and referral campaigns. This integration focuses on educating people about using Voucherify in composable commerce to enhance advanced promotions capabilities.
+
+## Integration architecture
+
+To extend the storefront by promotion capabilities, Voucherify integration:
+1. wrap commerce-generic service extending server API by additional methods and data related to marketing promotions,
+2. provides additional UI components that allow users to manage coupons and auto-applied promotions
+
+Voucherify commerce wrapper implements a standard checkout integration pattern, where for each cart update, the wrapper makes Validation (eligibility check, discount calculations) and Qualification (list of applicable promotions) requests to Voucherify REST API. Collected pieces of information extend cart data.
+
+  
+
+```mermaid
+sequenceDiagram
+  participant B as Browser
+  participant VCW as Commerce Wrapper
+  participant CGS as Commerce Generic
+  participant V as Voucherify API
+  B->>VCW: tRPC, CommerceServiceWithDiscounts interface extends CommerceService
+  activate VCW
+  VCW->>CGS: CommerceService interface (getCart, addCartItem etc)
+  activate CGS
+  CGS-->>VCW: Cart object
+  deactivate CGS
+  Note over VCW: Get cart vouchers codes from storage
+  VCW->>V: get available cart promotions
+  activate V
+  V-->>VCW: promotions
+  deactivate V
+  VCW->>V: validate promotions and vouchers codes
+  activate V
+  V-->>VCW: eligibility check and discount calculations results
+  deactivate V
+  Note over VCW: Extend cart by discounts details
+  VCW-->>B: CartWithDiscount object
+  deactivate VCW
+
+```
+
+## Reference Files
+
+### Backend files
+
+- `packages/voucherify`: This package contains services used to wrap commerce-generic services and communicate to the Voucherify REST API.
+
+### React Components
+
+- `composable-ui/src/components/forms/coupon-form.tsx`: Form to manage coupons, gift and loyalties cards
+- `composable-ui/src/components/cart/cart-promotions.tsx`: Display auto-applied cart promotions
+
+## Integrating Voucherify with Composable UI
+
+1. [Create a Voucherify account](https://app.voucherify.io/#/signup).
+2. In Voucherify Dashbioard, [set Discount Application Rule to "Partial"](https://support.voucherify.io/article/604-stacking-rules#application-rules)
```

Comment:
1. Spelling error in "Dashboard"
2. A screenshot or instructions of how to navigate to set up the "Partial" rule could help here,

![image](https://github.com/voucherifyio/composable-ui-fork/assets/13950845/68e0d553-2f1b-4986-b343-d69b2669bf1b)